src/quote/zeroEx/componentsQuoteProvider.ts

```diff
@@ -60,20 +58,18 @@ export class ComponentsQuoteProvider {
         const fakeResponse = this.getFakeZeroExResponse(amount)
         quotePromises.push(fakeResponse)
       } else {
-        const params = isMinting
-          ? {
-              buyToken,
-              sellToken,
-              buyAmount: buyAmount.toString(),
-              slippagePercentage,
-            }
-          : {
-              buyToken,
-              sellToken,
-              sellAmount: sellAmount.toString(),
-              slippagePercentage,
-            }
-        const quotePromise = zeroExApi.getSwapQuote(params, chainId ?? 1)
+        const params: SwapQuoteRequest = {
+          chainId,
+          outputToken: buyToken,
+          inputToken: sellToken,
+          slippage,
+        }
+        if (isMinting) {
+          params.outputAmount = buyAmount.toString()
+        } else {
+          params.inputAmount = sellAmount.toString()
+        }
+        const quotePromise = swapQuoteProvider.getSwapQuote(params)
```

Comment:
Using the new `SwapQuoteRequest` inside this components quote provider which is an essential part of the FlashMintZeroEx provider.